pkg/agent/controller/noderoute/node_route_controller.go

```diff
@@ -297,10 +297,20 @@ func (c *Controller) reconcile() error {
 }
 
 // Run will create defaultWorkers workers (go routines) which will process the Node events from the
-// workqueue.
+// work queue.
```

Comment:
workqueue is a widely used compound word in Kubernetes, even in linux.
https://github.com/kubernetes/kubernetes/blob/583d83a13557fc4ceb68355fa590c194f23a1721/staging/src/k8s.io/client-go/util/workqueue/delaying_queue.go#L32
https://github.com/torvalds/linux/blob/6f0d349d922ba44e4348a17a78ea51b7135965b1/Documentation/core-api/workqueue.rst#introduction
So maybe no need to separate them.

Reply:
done.